pytest_fixtures/component/satellite_auth.py

```diff
@@ -298,7 +298,7 @@ def enroll_idm_and_configure_external_auth(default_sat):
 def configure_realm():
     """Configure realm"""
     realm = settings.upgrade.vm_domain.upper()
-    run_command(cmd=f'echo "{settings.ipa.keytab}" > /root/freeipa.keytab')
+    run_command(cmd=f'wget -O /root/freeipa.keytab {settings.ipa.keytab_url}')
```

Comment:
As curl is installed by default and wget is not, did you try this with `curl` instead of `wget` here?